tests/functional/CSV/Safe/ExtractorTest.php

```diff
@@ -68,9 +70,16 @@ public function testExtractEmptyCSVFile(): void
 
         $extractor = new Extractor($this->reader, 0);
 
-        $this->assertDoesIterateLike(
+        $this->assertExtractorExtractsLike(
             [],
-            $extractor->extract()
+            $extractor
+        );
+    }
+
+    public function pipelineRunner(): PipelineRunnerInterface
+    {
+        return new \Kiboko\Component\Pipeline\PipelineRunner(
+            new NullLogger()
         );
     }
```

Comment:
Use a copy of this [PipelineRunner](https://github.com/php-etl/csv-flow/blob/master/tests/functional/PipelineRunner.php)